app/models/public-site.js

```diff
@@ -16,5 +16,6 @@ export default DS.Model.extend({
   facebookUrl: DS.attr('string'),
   blogUrl: DS.attr('string'),
   contactEmail: DS.attr('string'),
-  contactPhone: DS.attr('string')
+  contactPhone: DS.attr('string'),
+  enabled: DS.attr(),
```

Comment:
@sethphillips Since we don't have a way to really exclude the site at the moment, and I thought that people may want to still direct link to it I changed the name of this setting to `includeInIndex` since it's a little more explicit what the field is being used for. PRs are submitted to cablecast backend and ui for editing. We'll work on getting this deployed to sandbox.trms.com tomorrow.